pallets/automation-time/src/benchmarking.rs

```diff
@@ -496,4 +497,32 @@ benchmarks! {
 
 		schedule_notify_tasks::<T>(caller.clone(), vec![new_time_slot], T::MaxTasksPerSlot::get());
 	}: { AutomationTime::<T>::shift_missed_tasks(last_time_slot, diff) }
+
+	migration_add_schedule_to_task {
+		let v in 1..100;
+
+		use migrations::add_schedule_to_task::{AccountTasks as OldAccountTasks, AddScheduleToTask, OldAction, OldTask};
+		use frame_support::traits::OnRuntimeUpgrade;
+
+		for i in 0..v {
+			let account_id: T::AccountId = account("Account", 0, i);
+			let task_id_1 = Pallet::<T>::generate_task_id(account_id.clone(), vec![0]);
```

Comment:
did you mean to have a `task_id_2`?

Reply:
I did originally, but ended up removing it. Putting in two tasks per `v` completely screws up the benchmarking numbers.

Reply:
Updated the var names to remove confusion